rotkehlchen/chain/ethereum/modules/aave/constants.py

```diff
@@ -1,3 +1,40 @@
-CPT_AAVE_V1 = 'aave-v1'
-CPT_AAVE_V2 = 'aave-v2'
-AAVE_LABEL = 'Aave'
+from enum import Enum
+from typing import Final
+
+
+LIQUIDATION_CALL: Final = b'\xe4\x13\xa3!\xe8h\x1d\x83\x1fM\xbc\xcb\xcay\r)R\xb5o\x97y\x08\xe4[\xe3s5S>\x00R\x86'  # noqa: E501
+ENABLE_COLLATERAL: Final = b'\x00\x05\x8aV\xea\x94e<\xdfO\x15-"z\xce"\xd4\xc0\n\xd9\x9e*C\xf5\x8c\xb7\xd9\xe3\xfe\xb2\x95\xf2'  # noqa: E501
+DISABLE_COLLATERAL: Final = b'D\xc5\x8d\x816[f\xddK\x1a\x7f6\xc2Z\xa9{\x8cq\xc3a\xeeI7\xad\xc1\xa0\x00\x00"}\xb5\xdd'  # noqa: E501
+WITHDRAW: Final = b'1\x15\xd1D\x9a{s,\x98l\xba\x18$N\x89zE\x0fa\xe1\xbb\x8dX\x9c\xd2\xe6\x9el\x89$\xf9\xf7'  # noqa: E501
+
+
+class AaveLabel(Enum):
+    """Protocol label constants for the AAVE protocol."""
+    AAVE_LABEL: Final = 'Aave'
+    AAVE_V2_LABEL: Final = 'AAVE v2'
+    AAVE_V3_LABEL: Final = 'AAVE v3'
+
+
+class AaveCpt(Enum):
```

Comment:
If you wanted to use an enum, why not use a `StrEnum` here and below? https://docs.python.org/3/library/enum.html#enum.StrEnum